Engine/ScriptAnalyzer.cs

```diff
@@ -822,13 +822,13 @@ private void Initialize(
             // Ensure that rules were actually loaded
             if (rules == null || rules.Any() == false)
             {
+                string errorMessage = string.Format( CultureInfo.CurrentCulture, Strings.RulesNotFound);
+
                 this.outputWriter.ThrowTerminatingError(
                     new ErrorRecord(
-                        new Exception(),
-                        string.Format(
-                            CultureInfo.CurrentCulture,
-                            Strings.RulesNotFound),
-                        ErrorCategory.ResourceExists,
+                        new Exception(errorMessage),
```

Comment:
where this error is throw, are we sure the only reason could be missing rules @Tadas ?
Kicked off CI for now to see what tests say

Reply:
actually, on closer inspection of the diff, only 2 things are changing and they aren't of concern:
- errorMessage is passed into Exception instead of nothing, which is an improvement
- ErrorCategory is changed from ResourceExists to ResourceUnavailable, which I guess is a bit better but `ObjectNotFound` is maybe even better so will change it to that. This is the same category that `gci notExists` returns